docs/guides/networking/linode-network/ip-failover-bgp-frr/index.md

```diff
@@ -0,0 +1,300 @@
+    - **Hostname** (`[HOSTNAME]`): The hostname defined on the Compute Instance you are configuring (ex: `atl-bgp-1.example.com`).
+    - **Role** (`[ROLE]`): The role of this Compute Instance within your failover strategy.
+      - `primary`: All requests are routed to this Compute Instance, provided it is accessible.
+      - `secondary`: If the `primary` instance fails, all requests are routed to this Compute Instance, provided it is accessible.
+    - **Data center ID** (`[DC_ID]`): The ID of this data center as defined by the list below:
+        - Atlanta (USA): `4`
+        - Dallas (USA): `2`
+        - Frankfurt (Germany): `10`
+        - Fremont (USA): `3`
+        - London (UK): `7`
+        - Mumbai (India): `14`
+        - Newark (USA): `6`
+        - Singapore: `9`
+        - Sydney (Australia): `16`
+        - Tokyo (Japan): `11`
+        - Toronto (Canada): `15`
+
+1.  The template below should be used for the FRR configuration on Compute Instance within your IP failover setup. Ensure you replace any instances of `[SHARED_IP]`, `[HOSTNAME]`, `[ROLE]`, and `[DC_ID]` as outlined above. Store the template somewhere that you can easily access later. In a later step, you copy the contents of the template and paste them into the VTY interactive shell.
+
+      {{< file "~/ipfailover.conf">}}
+hostname [HOSTNAME]
+
+router bgp 65000
+no bgp ebgp-requires-policy
+coalesce-time 1000
+bgp bestpath as-path multipath-relax
+neighbor RS peer-group
+neighbor RS remote-as external
+neighbor RS capability extended-nexthop
+neighbor 2600:3c0f:[DC_ID]:34::1 peer-group RS
+neighbor 2600:3c0f:[DC_ID]:34::2 peer-group RS
+neighbor 2600:3c0f:[DC_ID]:34::3 peer-group RS
+neighbor 2600:3c0f:[DC_ID]:34::4 peer-group RS
+address-family ipv4 unicast
+  network [SHARED_IP]/32 route-map [ROLE]
+  redistribute static
+exit-address-family
+route-map primary permit 10
+set large-community 63949:1:1
+route-map secondary permit 10
+set large-community 63949:1:2
+{{</ file >}}
+
+1.  Run the VTY shell:
+
+        sudo vtysh
+
+1.  Enter configuration mode:
+
+        conf t
+
+1.  Copy the contents of your template configuration file and paste them into the VTY shell:
+
+1.  Tell the VTY shell that you are done entering your configurations:
+
+        end
+
+1. Write your configurations to VTY:
+
+        write
+
+1.  Verify that the configurations you entered were correctly written by showing VTY's running configuration:
+
+        show running-config
+
+1.  Exit out of the VTY shell:
+
+        q
+
+## Configure the Network Interface
+
+1.  Configure the Compute Instance's network interface as detailed below. Replace `[SHARED_IP]` with the Shared IP address you've configured.
+
+    - **Debian 10 & Ubuntu 18.04**
+
+        Edit the `/etc/network/interfaces` file with the following entries.
+
+        {{< file >}}
+up   ip addr add [SHARED_IP]/32 dev eth0 label eth0
```

Comment:
in theory the official way here would be to just repeat the 
```
iface eth0 inet static
   address [SHARED_IP]/32
```
section for all IPs but this works too